include/libtorrent/torrent_handle.hpp

```diff
@@ -521,7 +522,7 @@ namespace libtorrent
 		// if the metadata is successfully set on the torrent, and false
 		// otherwise. If the torrent already has metadata, this function will not
 		// affect the torrent, and false will be returned.
-		bool set_metadata(char const* metadata, int size) const;
```

Comment:
would you mind preserving the existing signature, under `#ifndef TORRENT_NO_DEPRECATE`, and with the `TORRENT_DEPRECATED` attribute?